### PR TITLE
fix: header prepaid plan buy credits

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Fix when user has a prepaid account and want to buy credits redirect to buy credits page and not show popup

### CODE REVIEW

File: src/components/Header/HeaderUserMenu/HeaderUserMenu.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ class HeaderUserMenu extends React.Component {
                 <strong>{user.plan.remainingCredits}</strong> {user.plan.description}
               </p>
             </div>
-            {user.hasClientManager && user.plan.buttonUrl && !user.plan.pendingFreeUpgrade ? (
+            {!user.hasClientManager && user.plan.buttonUrl && !user.plan.pendingFreeUpgrade ? (
```
Comment (andresmoschini) [inline src/components/Header/HeaderUserMenu/HeaderUserMenu.js]: Did you verify other scenarios?

Maybe, before merging it, is a good idea waiting for some "Eagle Eyes" @CarlosSampedro's tests.

@fcoronelmakingsense could you add the links to testing this PR in different environments?

**Comment by fcoronelmakingsense** [comment]
To Try
• https://cdn.fromdoppler.com/doppler-webapp/build710/#/reports (PROD)
• https://cdn.fromdoppler.com/doppler-webapp/int-build710/#/reports (INT)
• https://cdn.fromdoppler.com/doppler-webapp/qa-build710/#/reports (QA)

File: src/components/Header/HeaderUserMenu/HeaderUserMenu.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/components/Header/HeaderUserMenu/HeaderUserMenu.js]: im already put in comment, thanks!!

**Comment by fcoronelmakingsense** [comment]
update build https://cdn.fromdoppler.com/doppler-webapp/build716/#/reports

**Comment by CarlosSampedro** [comment]
Hi @fcoronelmakingsense 

I have reported the following bug related to the PR:

http://jira.makingsense.com/browse/DBR-307

Regards
Carlos

**Pull request merged**